frontend/src/Series/Index/Table/SeriesIndexRow.js

```diff
@@ -413,6 +414,22 @@ class SeriesIndexRow extends Component {
               );
             }
 
+            if (name === 'releaseGroups') {
+              const joinedReleaseGroups = releaseGroups.join(', ');
+              const truncatedReleaseGroups = releaseGroups.slice(0, 4).join(', ');
```

Comment:
I think if this truncates we should show an ellipsis as well and I think we'd want to cap at 3 to deal with longer release groups:

```
const truncatedReleaseGroups = releaseGroups.length > 3 ?
  `${releaseGroups.slice(0, 3).join(', ')}...` : 
  joinedReleaseGroups;
```